### ISSUE TITLE
Типограф

### ISSUE THREAD

**Issue by loveorigami** [skeeks-semenov/yii2-ckeditor#1]
Не планируете добавить типограф в плагины?
Например, http://mdash.ru/

**Comment by skeeks-semenov** [comment]
@loveorigami А есть ли уже готовый плагин для ckeditor?

**Comment by loveorigami** [comment]
Я так понял, это заготовка под типограф
https://github.com/evgeniy-trebin/ckeditor-plugin-typograf

Внутри можно реализовать обработку выделенного текста через один из типографов.

Для TinyiMce у меня стоит этот - https://habrahabr.ru/post/67687/

Если посмотреть код, то там вполне понятно, как он работает без Api

**Comment by skeeks-semenov** [comment]
Да, наверное, это будет достаточно полезный плагин. В свободное время посмотрю, подключу.

**Comment by loveorigami** [comment]
Этот можно прикрутить )
https://github.com/vgrish/ckeditor-plugin-typograf
